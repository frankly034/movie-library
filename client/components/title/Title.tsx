"use client";

import styled from "styled-components";

type TitleProps = {
  textColor?: string;
  size?: string;
};

const Title = styled.h1<TitleProps>`
  color: ${(props) => props.textColor || "#f9f9f9"};
  font-size: ${(props) => props.size || "16px"};
`;

export default Title;
