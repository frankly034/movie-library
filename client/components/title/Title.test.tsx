import { render } from "@testing-library/react";

import Title from "./Title";

describe("Component - Title", () => {
  it("renders title component", () => {
    const { container } = render(<Title>{"Gift a friend this holiday"}</Title>);
    expect(container).toMatchSnapshot();
  });
});
