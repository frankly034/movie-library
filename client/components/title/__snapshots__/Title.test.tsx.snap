// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Title renders title component 1`] = `
<div>
  <h1
    class="sc-bcPKhP gGePeu"
  >
    Gift a friend this holiday
  </h1>
</div>
`;
